app/views/all_service_transactions/filter.njk

```diff
@@ -0,0 +1,217 @@
+
+<div class="transactions-filter govuk-grid-row">
```

Comment:
Just a reminder to myself (as discussed), if any of this filter/ table template was re-usable between the transactions and all transactions page (hopefully all of it with a condition or two for specific changes) it would mean we don't have to maintain two different pages. Looking good so far!